CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,47 @@
+# Code of Conduct
+
+## Conduct
+
+Contact: a moderator (see below), or a member of the community you feel you can trust.
+
+* We are committed to providing a friendly, safe, and welcoming environment for all,
+ regardless of level of experience, gender, gender identity and expression, sexual orientation,
+ disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
+ * Please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
+ * Please be kind and courteous. There's no need to be mean or rude.
+ * Respect that people have differences of opinion and that every design or implementation
+ choice carries a trade-off and numerous costs. There is seldom a right answer.
+ * Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+ * We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In
```

Comment:
Just in case anyone else spots the `http:`, no https://citizencodeofconduct.org/ doesn't work.

Reply:
https://github.com/stumpsyn/policies/issues/8

Reply:
Right now I can't load the site over HTTP either. It looks like this site isn't very reliable, https://github.com/rust-lang/rust/issues/57402 is a similar report.

I suggest we link either https://web.archive.org/web/20200213153345/http://citizencodeofconduct.org/#unacceptable-behavior or https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md#4-unacceptable-behavior instead.